src/main/kotlin/org/jetbrains/kotlinx/publisher/PublicationsExtension.kt

```diff
@@ -392,8 +392,11 @@ class PublicationsExtension(private val project: Project) {
         private const val SONATYPE_REPOSITORY_NAME = "Sonatype"
         private const val PUBLISH_TO_SONATYPE_WITH_EXCLUDING_TASK = "publishToSonatypeWithExcluding"
         private const val PUBLISH_TO_SONATYPE_AND_RELEASE_TASK = "publishToSonatypeAndRelease"
-        private const val CLOSE_AND_RELEASE_TASK = "closeAndReleaseStagingRepository"
+        private const val CLOSE_AND_RELEASE_TASK = "closeAndReleaseStagingRepositories"
```

Comment:
Does new name come from new sonatype publishing plugin?

Reply:
Yes, it was changed. See https://github.com/gradle-nexus/publish-plugin/pull/285/files